lib/msal-node/docs/initialize-confidential-client-application.md

```diff
@@ -38,7 +39,7 @@ const clientConfig = {
             thumbprint: process.env.thumbprint,
             privateKey: process.env.privateKey,
         }, // OR
-        clientAssertion: "assertion",
+        clientAssertion: assertionCallback,
```

Comment:
As stated before, both of these must be supported. Ideally you can reuse the "clientAssertion" key, but if not possible, it's acceptable to have an additional "clientAssertionCallback: my_callback"